server/tox.ini

```diff
@@ -1,14 +1,11 @@
 [tox]
-envlist = pep8,py36-inmiso4-stable,py39-inm{master,iso6-stable,iso5-stable,latest}
+envlist = pep8,py39-inm{master,iso6-stable,latest}
 skip_missing_interpreters=True
 requires =
     pip >= 21.0.1
     virtualenv<20.22.0
     wheel
 
-[testenv:py36]
-basepython=python3.6
-
 [testenv:py39]
 basepython=python3.9
```

Comment:
Might as well delete this too because it's completely redundant (tox infers it from the `py39-*` name).